CanvasApi.Client.Test/CanvasApi.Client.Test.csproj

```diff
@@ -24,7 +24,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Update="appsettings-test.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>

```

Comment:
We will actually be removing this file in when I do my PR. i have a better way to handle this scenario using user-secrets that is build into dotnet.